seed/seedRole.js

```diff
@@ -0,0 +1,49 @@
+const Role = require('~/models/role')
+const {
+  roles: { STUDENT, MENTOR, ADMIN, SUPERADMIN }
+} = require('~/consts/auth')
+const logger = require('~/logger/logger')
+
+const SeedRole = {
+  createStudentRole: async () => {
+    try {
+      const studentRole = await Role.create({ value: STUDENT })
+
+      return studentRole
+    } catch (err) {
+      logger.error(err)
+    }
+  },
+
+  createMentorRole: async () => {
+    try {
+      const mentorRole = await Role.create({ value: MENTOR })
+
+      return mentorRole
+    } catch (err) {
+      logger.error(err)
+    }
+  },
+
+  createAdminRole: async () => {
+    try {
+      const adminRole = await Role.create({ value: ADMIN })
+
+      return adminRole
+    } catch (err) {
+      logger.error(err)
+    }
+  },
+
+  createSuperAdminRole: async () => {
+    try {
+      const superAdminRole = await Role.create({ value: SUPERADMIN })
+
+      return superAdminRole
+    } catch (err) {
+      logger.error(err)
+    }
+  }
+}
+
```

Comment:
In my opinion, this is _too_ much duplicate code. The only value that changes is the `value` property value you pass in. Maybe make a separate function (for creating the role), and reuse it in all the other four (by passing the value parameter into the created function)?

Reply:
good catch, thanks!